tests/Unit/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/DampedHarmonicGaugeQuantities.py

```diff
@@ -0,0 +1,204 @@
+# Distributed under the MIT License.
```

Comment:
Remove `GaugeQuantities` from the name please, to make these test file names correspond to the file names that are being tested